PortFxCop/src/System.Runtime.Analyzers/Core/AvoidUnsealedAttributes.Fixer.cs

```diff
@@ -10,27 +8,49 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using Analyzer.Utilities;
 
 namespace System.Runtime.Analyzers
-{                              
+{
     /// <summary>
     /// CA1813: Avoid unsealed attributes
     /// </summary>
-    public abstract class AvoidUnsealedAttributesFixer : CodeFixProvider
+    [ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.VisualBasic), Shared]
+    public class AvoidUnsealedAttributesFixer : CodeFixProvider
     {
         public sealed override ImmutableArray<string> FixableDiagnosticIds => ImmutableArray.Create(AvoidUnsealedAttributesAnalyzer.RuleId);
 
-        public sealed override FixAllProvider GetFixAllProvider()
+        public override async Task RegisterCodeFixesAsync(CodeFixContext context)
         {
-            // See https://github.com/dotnet/roslyn/blob/master/docs/analyzers/FixAllProvider.md for more information on Fix All Providers
-            return WellKnownFixAllProviders.BatchFixer;
```

Comment:
Was FixAll support disabled on purpose here? Is it broken or seems not useful for this fix?